node_lambnik/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '2.1'
```

Comment:
As on the breakable toy, I think it makes sense to start out with a more up-to-date docker-compose config version.  `3.1` seems reasonable, since only very recent versions support newer minor versions.